test/integration/test-fixtures/image-os-binary-overlap/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM alpine:latest@sha256:7144f7bab3d4c2648d7e59409f15ec52a18006a128c733fcff20d3a4a54ba44a
```

Comment:
Another sha + `latest`. (Sorry for not marking all of them in the first round of comments.)

I wonder if it makes sense to just us `alpine@sha...`? (Maybe I should have done that in #2137 as well)

Reply:
I think it's ok to specify the version here - it doesn't make a different on the Docker side of things and communicates more information to future reads what the exact version is that this sha pins to so they don't have to hunt for that.